# As far as I (I=aacid@kde.org) know this is not upstream (may be needed in the combinations of mingw+Qt they support)
# Just enough to make it
--- a/src/corelib/io/qfilesystemengine_win.cpp	2022-01-14 21:46:24.091832900 +0100
+++ b/src/corelib/io/qfilesystemengine_win.cpp	2022-01-14 21:56:48.142356500 +0100
@@ -664,7 +664,7 @@
     return QFileSystemEntry(ret, QFileSystemEntry::FromInternalPath());
 }
 
-#if defined(Q_CC_MINGW) && WINVER < 0x0602 //  Windows 8 onwards
+#if !defined(QT_BOOTSTRAPPED) && !defined(QT_BUILD_QMAKE) && defined(Q_CC_MINGW) && WINVER < 0x0602 //  Windows 8 onwards
 
 typedef struct _FILE_ID_INFO {
     ULONGLONG VolumeSerialNumber;
